index.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
+<html lang="en" data-theme="light">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Document</title>
+		<link href="https://cdn.jsdelivr.net/npm/daisyui@3.5.1/dist/full.css" rel="stylesheet" type="text/css" />
+		<script src="https://cdn.tailwindcss.com"></script>
+
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+	</head>
+	<body class="min-w-screen min-h-screen flex md:flex-row flex-col bg-slate-100">
+		<div class="flex card card-side md:w-full shadow-lg overflow-hidden p-5 bg-white md:m-10 m-5 box-border">
+			<div
+				class="overflow-hidden w-1/4 hidden bg-purple-800 md:flex flex-col justify-between card py-14 px-10 text-white text-lg">
+				<div class="flex flex-col gap-6 z-10">
+					<h2 class="card-title text-4xl">Contact Information</h2>
+					<p class="font-normal text-purple-400">Fill up the form and our Team will get back to you within 24 hours.</p>
+				</div>
+
+				<div class="flex flex-col my-5 gap-14 z-10">
+					<div class="flex items-center gap-3">
+						<span class="material-symbols-outlined"> call </span> 48 780 182 501
+					</div>
+					<div class="flex items-center gap-3">
+						<span class="material-symbols-outlined"> mail </span> hello@flowbase.com
+					</div>
+					<div class="flex items-center gap-3">
+						<span class="material-symbols-outlined"> location_on </span> 102 Street 2714 Don
+					</div>
+				</div>
+
+				<div class="flex items-center justify-around w-1/2 z-10">
+					<svg xmlns="http://www.w3.org/2000/svg" height="1em" viewBox="0 0 320 512">
+						<!--! Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. -->
```

Comment:
po co ci komentarze w kodzie?

Reply:
Myślałem że to potrzebne do legalnego użycia ikon z fontawsome. Jeśli to nie prawda to usunę te komentarze.